scripts/docker-server.sh

```diff
@@ -0,0 +1,83 @@
+#! /bin/sh
+
+port="9090"
+docker_image="learn-ocaml-docker"
+container_name="learn-ocaml-docker-container"
+
+function print_usage() {
+    printf "Usage: %s COMMAND <OPTIONS>\n\
+Commands:\n\
+  init  \t Initializes the server and runs it.\n\
+  start \t Starts an already initialized server.\n\
+  restart\t Restarts the current server (starts it if it is not running).\n\
+  stop  \t Stops the server.\n\
+  remove\t Stops the server and removes it from the existing containers.\n\
+Options:\n\
+  -port (default = 9090): Port used to communicate with the server.\n\
+  -docker-image (default = learn-ocaml-docker): name of the generated\
+  docker image.\n\
+  -container-name (default = learn-ocaml-docker-container): name of the\
+  container in docker container system.\n" "$0"
+}
+
+function run_image () { # port, container, image
+    docker run -d \
+           -p "$1":9090 \
+           --name "$2" "$3"
+    
+}
+
+while [[ $# -gt 0 ]]; do
+  curr="$1"
+  case $curr in
+      -port)
+      port="$2"
+      shift
```

Comment:
I think that a sequence of ``shift`` can be replaced by ``shift 2``.

Reply:
Indeed, that's fixed now.